### PR TITLE
Beta version of The Room game

### PR DESCRIPTION
**Pull request by DMyTryC**
This is the beta version of the room game

### CODE REVIEW

**Comment by schwabdidier** [comment]
Still a problem

**Comment by YannRobert** [comment]
Please do not change end of lines in file multilinguism.csv
The diff should clearly show only the lines you added/removed

You changed EOL from LF to CRLF. Please make it LF again.

File: gazeplay-data/src/main/resources/data/multilinguism/multilinguism.csv [hunk 1]
```diff
@@ -38,16 +38,15 @@ Biboules,Biboules,Biballs,Biboles,Biboules
 Robots,Robots,Robots,Roboters,Robots
 Memory,Memory,Memory,Memory,Memory
 EnableRewardSound,Activer le son « bravo »,Enable Reward sound,Aktiviere den Sound Bravo,Aktivigu la sonon Bravo
-MenuOrientation,Orientation du menu (nécessite de relancer le jeu),Menu Orientation (restart game to take effect),Menüorientierung (erfordert Neustart des Spiels),Orientación del menú (requiere reiniciar el juego)
```
Comment (YannRobert) [inline gazeplay-data/src/main/resources/data/multilinguism/multilinguism.csv]: line remove, why?

File: gazeplay-data/src/main/resources/data/multilinguism/multilinguism.csv [hunk 2]
```diff
@@ -38,16 +38,15 @@ Biboules,Biboules,Biballs,Biboles,Biboules
...
-OpenMemory,Memory ouvert,OpenMemory,OpenMemory,OpenMemory
+OpenMemory,Vise les pairs !,OpenMemory,Fokus auf Peers !,OpenMemory
 Video Player with Feedback,Lecteur Video avec retour,Video Player with Feedback,Video Player mit Feedback,Reproductor de video con comentarios
 Choose Game Variante,Choisi une variante,Choose Variante,Wählen Sie Variante,Elija Variante
 Enter FullScreen,Mettre en Plein écran,Enter FullScreen,Vollbild,Pantalla completa
 Exit FullScreen,Sortir du Plein écran,Exit FullScreen,Beenden Sie den Vollbildmodus,Salir de pantalla completa
 Scribble,Gribouille,Scribble,Kritzeln,Garabato
 Lapins,Lapins,Rabbits,Conejos,Kaninchen
 Divisor,Diviseur,Divisor,Divisor,Divisor
-Piano,Piano,Piano,Piano,Piano
```
Comment (YannRobert) [inline gazeplay-data/src/main/resources/data/multilinguism/multilinguism.csv]: 2 lines removed, why?

Comment (DMyTryC) [reply gazeplay-data/src/main/resources/data/multilinguism/multilinguism.csv]: This was a mistake I did when changing the file, solving it and repushing

File: gazeplay-data/src/main/resources/data/multilinguism/multilinguism.csv [hunk 3]
```diff
@@ -41,7 +41,7 @@ EnableRewardSound,Activer le son « bravo »,Enable Reward sound,Aktiviere den
 MenuOrientation,Orientation du menu (nécessite de relancer le jeu),Menu Orientation (restart game to take effect),Menüorientierung (erfordert Neustart des Spiels),Orientación del menú (requiere reiniciar el juego)
 Music,Musique,Music,Musik,Musica
 Volume,Volume,Volume,Volume,Volume
-OpenMemory,Memory ouvert,OpenMemory,OpenMemory,OpenMemory
+OpenMemory,Vise les pairs !,OpenMemory,Fokus auf Peers !,OpenMemory
```
Comment (YannRobert) [inline gazeplay-data/src/main/resources/data/multilinguism/multilinguism.csv]: are you sure for this? I think this was recently changed already.
this change is not related to the subject of the pull request.
may need to propose this change in a separate pull request.

Comment (schwabdidier) [reply gazeplay-data/src/main/resources/data/multilinguism/multilinguism.csv]: I changed to OpenMemory and "Memory ouvert" before last release. Please @DMyTryC avoid to revert what has been done by others and next time, don't forget to update your code before pull request.

Comment (schwabdidier) [reply gazeplay-data/src/main/resources/data/multilinguism/multilinguism.csv]: before starting to work on your code would be better.

**Comment by DMyTryC** [comment]
Recreated the game from the new files, closing this pull request

**Comment by YannRobert** [comment]
Please avoid to close a PR to recreate it with the same code.
If this is just in order to merge, rebase, squash, just to it in a local branch, recreate your local branch, then ```push --force``` to the same remote branch on github.

If you do recreate a PR anyways, always reference link the new and old PR in a comment.
